### ISSUE TITLE
Add options to customize

### ISSUE THREAD

**Issue by gavinhungry** [gavinhungry/seasurf#3]
``` javascript
app.use(seasurf({
  token: function(req, res) {
    // default only checks X-CSRF-Token
    return req.headers['my-custom-csrf-header'];
  },

  fail: function(req, res) {
    res.status(403).json({ error: 'Your CSRF token is bogus!' });
  }
})
```

**Comment by gavinhungry** [comment]
Or maybe `CSRF-Token` should be the default

**Comment by gavinhungry** [comment]
Fixed by 77a11c18c62e203cbddcb3461bd8f11b525153a2
